src/main/java/org/sonarlint/intellij/ui/tree/TreeModelBuilder.java

```diff
@@ -47,21 +50,14 @@
  * Should be optimize to minimize the recreation of portions of the tree.
  */
 public class TreeModelBuilder {
+  private static final List<String> SEVERITY_ORDER = ImmutableList.of("BLOCKER", "CRITICAL", "MAJOR", "MINOR", "INFO");
+  private final static Comparator<IssuePointer> ISSUE_COMPARATOR = new IssueComparator();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)